PolyEngine/Engine/Src/InputWorldComponent.cpp

```diff
@@ -17,3 +17,57 @@ bool Poly::InputWorldComponent::IsPressed(const std::initializer_list<eMouseButt
 		result = result && CurrMouseButton[i];
 	return result;
 }
+
+bool Poly::InputWorldComponent::IsPressed(size_t playerID, eControllerButton button) const
+{
+	ASSERTE(IsControllerConnected(playerID), "Controller not connected!");
+	i32 joystickID = PlayerIDToJoystickID[playerID].Value();
```

Comment:
After further consideration joystickID should be kept in `size_t` variable type. This will also fix Windows compilation issues.